tests/DS/test_ds_misc.sh

```diff
@@ -242,7 +242,7 @@ test_run "sds_external_xccdf" test_sds_external_xccdf sds_external_xccdf sds_ext
 test_run "sds_tailoring" test_sds_tailoring sds_tailoring sds_tailoring/sds.ds.xml scap_com.example_datastream_with_tailoring xccdf_com.example_cref_tailoring_01 xccdf_com.example_profile_tailoring
 
 test_run "eval_simple" test_eval eval_simple/sds.xml
-test_run "cpe_in_ds" test_eval cpe_in_ds/sds.xml
+test -f "/etc/sysconfig/prelink" && test_run "cpe_in_ds" test_eval cpe_in_ds/sds.xml
```

Comment:
It's better to `return 255` for this test if it is being skipped so that CTest could be able to catch it. Like in `require rpm || return 255`